core/src/core-api/globalsearch.js

```diff
@@ -64,13 +64,17 @@ class LuigiGlobalSearch {
    * @memberof LuigiGlobalSearch
    * @example Luigi.globalSearch().getSearchString()
    */
-  getSearchString() {}
+  getSearchString() {
+    return Luigi.getGlobalSearchString();
```

Comment:
Could it happen that this function is called to early? and should we do something about it?  .. I remember other functions that only work after Luigi init. But for search it should be obvious that you can only use it when Luigi is there. So could be fine like that.

Reply:
Yes, I think so too and should be fine. If you want to use Luigi global search feature  it should be obvious that Luigi is there.